core/src/main/java/org/apache/accumulo/core/client/summary/Summary.java

```diff
@@ -78,27 +80,38 @@ public long getLarge() {
     }
 
     /**
+     * @return The number of files that were deleted after the summary retrieval operations started. This is a rare race condition where a compaction causes a
+     *         file to be deleted while retrieving summaries. When this happens, the file that replaced the deleted file can not be used because it may contain
+     *         duplication summary information for other files. Avoiding this race condition would be expensive, so reporting it was chosen. If this condition
+     *         must be avoided, then compactions must be stopped. Compactions could be stopped on a cloned table to avoid this.
+     */
+    public long getDeleted() {
+      return deleted;
```

Comment:
Is this onerous to compute? If this is rare and there is a workaround, I could see keeping this from public API. If it's not bad to compute/track, it's not an issue.